credmark/model/ledger/ledger.py

```diff
@@ -144,3 +149,32 @@ def get_erc20_transfers(self, columns: List[str],
                                      TokenTransferTable,
                                      columns, where, group_by,
                                      order_by, limit, offset)
+
+    def get_block_number_of_date(self, dt_in: Union[datetime, date]):
+        txn_blocks = self.Block.Columns
+        dt = datetime(dt_in.year, dt_in.month, dt_in.day, tzinfo=timezone.utc)
+        res = self.get_blocks(
```

Comment:
use model rpc.get_blocknumber

Reply:
moved to block_number and use rpc.get_blocknumber for date -> block_number.